crates/tests/tests-common/src/request.rs

```diff
@@ -153,3 +153,42 @@ async fn make_request<Response: for<'a> serde::Deserialize<'a>>(
         )
     })
 }
+
+/// Generate/Update static/query.schema.json
+/// this test should be ignored unless explicitly invoked
+#[ignore]
+#[test]
+fn generate_query_request_schema() {
```

Comment:
Added these utilities to generate query and mutation request schemas, to validate tests files. These tests are not invoked unless explicitly called upon. There's probably better ways to do this, and we can remove them if there's any feelings against keeping them around.